README.md

```diff
@@ -18,15 +18,19 @@ or from source:
 pip install .
 ```
 
-(`python setup.py install` will no longer work)
-
 or with conda (see details below)
 
 ```
 conda config --add channels conda-forge
 conda install pyamg
 ```
 
```

Comment:
Yeah, this is a little clunky, and we will have to remember to edit this line and the line in `docstrings.yml`  whenever  the list of packages changes.  But from your conversation a while ago with jakirkham, it looks like this is needed.